tests/fields/test_resolvers.py

```diff
@@ -47,6 +50,52 @@ def name(self) -> str:
     assert definition.fields[0].base_resolver(None) == Query().name()
 
 
+def test_staticmethod_resolver_fields():
+    @strawberry.type
+    class Query:
+        @strawberry.field
+        @staticmethod
+        def name() -> str:
+            return "Name"
+
+    definition = Query._type_definition
+
+    assert definition.name == "Query"
+    assert len(definition.fields) == 1
+
+    assert definition.fields[0].python_name == "name"
+    assert definition.fields[0].graphql_name is None
+    assert definition.fields[0].type == str
+    assert definition.fields[0].base_resolver() == Query.name()
+
+    assert Query.name() == "Name"
+    assert Query().name() == "Name"
+
+
+def test_classmethod_resolver_fields():
+    @strawberry.type
+    class Query:
+        my_val: ClassVar[str] = "thingy"
+
+        @strawberry.field
+        @classmethod
+        def val(cls) -> str:
+            return cls.my_val
```

Comment:
shall we add tests for staticmethods and classmethods defined on other classes? I know they work, but it is worth testing them 😊

Reply:
We already have one for `classmethod`s:
 https://github.com/strawberry-graphql/strawberry/blob/c5a1b6154810d2c186d916346d1ef77578060fc4/tests/schema/test_resolvers.py#L205

Reply:
let's do one for static methods too :D